docs/content/basic-concepts.md

```diff
@@ -15,7 +15,7 @@ Check it out [here](https://github.com/FormidableLabs/spectacle-boilerplate/blob
 ```jsx
 // index.js
 
-import React, { Component } from 'react';
+import React, { Component } from "react";
```

Comment:
My IDE has been known to force this opinion where it isn't applicable or necessary to change, that might be the case here, _or_ it could be the fault of the conflicting eslint configs. 🤔